### PR TITLE
x11-misc/revelation: version bump to 0.55

### PR DESCRIPTION
**Pull request by jayofdoom**
New dependency: defusedxml
https://github.com/mikelolasagasti/revelation/commit/edb67aa58d42a2075d2d8c132817eacfdb39a916

Includes upstream meson.build fix, removing need for
patch.

Package-Manager: Portage-3.0.30, Repoman-3.0.3

### CODE REVIEW

File: x11-misc/revelation/revelation-0.5.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline x11-misc/revelation/revelation-0.5.5.ebuild:4]: 8 work?

File: x11-misc/revelation/revelation-0.5.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	')
+	x11-libs/gtk+:3
+	dev-libs/glib
+	dev-libs/gobject-introspection
+"
+
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	find -name '*.py' -exec \
```
Comment (thesamesam) [inline x11-misc/revelation/revelation-0.5.5.ebuild:38]: It'd be nice to patch this upstream, e.g. https://github.com/p0dalirius/pydsinternals/commit/05904ec91b230150c04ba10bb4556450eead5438.

Comment (jayofdoom) [reply x11-misc/revelation/revelation-0.5.5.ebuild:37]: I looked into this, and honestly am not sure I want to do this right now. If I were in upstream's shoes, I wouldn't want to commit to supporting both libraries unless I ran CI for it... which makes me not want to ask someone else to accept a patch to do just that :).

File: x11-misc/revelation/revelation-0.5.5.ebuild (hunk 1 above)
Comment (jayofdoom) [reply x11-misc/revelation/revelation-0.5.5.ebuild]: With some work to migrate to the EAPI 8 version of xdg.

File: x11-misc/revelation/revelation-0.5.5.ebuild (hunk 2 above)
Comment (thesamesam) [reply x11-misc/revelation/revelation-0.5.5.ebuild:37]: Well, the point is it's trivial, and it's a simple fallback. But alright.

Comment (jayofdoom) [reply x11-misc/revelation/revelation-0.5.5.ebuild:37]: I'm just a little peculiar about making changes that impact crypto things unless I'm :100: sure that it's not doing anything problematic (like reducing the test coverage). If I don't have the time to do the work to ensure upstream tests with both libraries, I don't want to be responsible for introducing that change upstream.

**Pull request closed without merging**